README.md

```diff
@@ -1,27 +1,49 @@
-# README
+# This application displays joint stock companies from Poland. Here you can see data such as stock indices, shareholder list, etc.
```

Comment:
I think it would look nicer this way:

```suggestion
# JSCLook 

This application displays joint stock companies from Poland. Here you can see data such as stock indices, shareholder list, etc.
```